haskell/experimental/private/module.bzl

```diff
@@ -338,6 +350,7 @@ def _build_haskell_module(
                     narrowed_deps_info.deps_hs_libraries,
                     narrowed_deps_info.empty_hs_libraries,
                     object_inputs,
+                    depset(static_libs + dynamic_libs),
```

Comment:
We've added this because of the error in PR description. However this problems doesn't happen for `zlib` so we are not sure if we are correctly including the `libphonenumber` library.  Does adding  dependencies to `run_ghc` here make sense from you perspective @googleson78  ?